__tests__/endpoints.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const allData = require("../db/data/test-data");
+
+beforeEach(() => seed(allData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200, an array of topic objects, each of which should have a 'slug' and a 'description' property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toEqual(expect.any(Array));
+        expect(Object.keys(response.body.topics[0])).toEqual(
+          expect.arrayContaining(["slug", "description"])
+        );
+      });
+  });
+  test("GET:400, sends an appropriate error message when given an bad request", () => {
```

Comment:
As this error isn't specific to the /api/topics endpoint, I'd separate it out into its own describe block.